source/common/config/subscription_factory_impl.cc

```diff
@@ -56,19 +56,18 @@ SubscriptionPtr SubscriptionFactoryImpl::subscriptionFromConfigSource(
           std::make_shared<Config::GrpcMuxImpl>(
               local_info_,
               Utility::factoryForGrpcApiConfigSource(cm_.grpcAsyncClientManager(),
-                                                     api_config_source, scope)
+                                                     api_config_source, scope, true)
```

Comment:
I'm not particularly familliar with this code, but by my reading, disabling the check here means that the subscriptions created using this AsyncClientImpl will throw an EnvoyException when their `start` methods are called if the configured cluster doesn't exist. In at least the RDS case this happens inside the `Init::TargetImpl` and it's not clear to me where that exception is handled. Are there tests that cover subscriptions throwing during start?

Reply:
We are disabling here because it is actually checked https://github.com/envoyproxy/envoy/blob/master/source/common/config/subscription_factory_impl.cc#L39 at the start of switch. So it is no different from previous implementation except that we now eliminate the second redundant check.